documentation/release_notes.rst

```diff
@@ -8,6 +8,60 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2021.4
+=============
+
+New Features
+------------
+-  Added the range-based versions of the following algorithms: ``any_of``, ``adjacent_find``,
```

Comment:
`of` --> `for`?

Reply:
both work

Reply:
We always use "of". Let's keep it consistent.